Voice Note/Voice_NoteApp.swift

```diff
@@ -6,19 +6,52 @@
 //
 
 import SwiftUI
-
+import CoreData
 
 @main
 struct Voice_NoteApp: App {
     @StateObject var speechRecognizer = SpeechRecognizer()
-    @StateObject var voiceNoteViewModel = VoiceNoteViewModel()
+    @StateObject var voiceNoteViewModel: VoiceNoteViewModel
     @StateObject var mapViewModel = MapViewModel()
 
+    // Core Data stack
+    let persistentContainer: NSPersistentContainer = {
+        let container = NSPersistentContainer(name: "VoiceNote")
```

Comment:
persistentContainer is set up and called here. I think this should be the one we use mostly, right?. 😄 I am just asking out of curiosity so please don't mind if I ask too many questions lol. Thanks 😂

Reply:
> persistentContainer is set up and called here. I think this should be the one we use mostly, right?. 😄 I am just asking out of curiosity so please don't mind if I ask too many questions lol. Thanks 😂

In my understanding, yes